Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -972,7 +996,25 @@ protected class ClassWriter : IClassWriter {
     }
 
     protected override void EmitConversionExpr(ConversionExpr e, bool inLetExprBody, ConcreteSyntaxTree wr, ConcreteSyntaxTree wStmts) {
+      var (pre, post) = ("", "");
+      if (e.E.Type.IsNumericBased(Type.NumericPersuasion.Int) || e.E.Type.IsBitVectorType) {
+        if (e.ToType.IsNumericBased(Type.NumericPersuasion.Real)) {
+          (pre, post) = ("_dafny.BigRational(", ", 1)");
+        } else if (e.ToType.IsCharType) {
+          (pre, post) = ("chr(", ")");
```

Comment:
Is `chr` the right function? (Does Dafny's 16-bits thing line up with Python's `chr`)?

Reply:
It should be fine, as `char`s can't overflow.